examples/react-emotion-ts-parcel/src/index.html

```diff
@@ -1,8 +1,8 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta name="viewport" content="width=device-width,initial-scale=1" />
-    <title>React Emotion TS Parcel</title>
+<!DOCTY'@onfido/castor/dist/castor.css';
```

Comment:
That's probably a mistake?

Reply:
Yup, nice catch, I don't know how the build passed 🤔